src/modules/shared/services/http.ts

```diff
@@ -0,0 +1,82 @@
+enum METHODS {
+  GET = 'get',
+  POST = 'post',
+  PUT = 'put',
+  DELETE = 'delete'
+}
+
+type stringObj = {[key:string]: string};
+
+interface Options {
+  timeout: number;
+  method: METHODS;
+  data: {[key:string]: any} | undefined;
+  headers: {[key:string]: string} | undefined;
```

Comment:
```suggestion
  timeout?: number;
  method?: METHODS;
  data?: {[key:string]: any};
  headers?: {[key:string]: string};
```
Поле method должно быть необязательным, потому что мы определяет его внутри